widget/embedded/src/containers/WidgetInfo/WidgetInfo.helpers.ts

```diff
@@ -11,9 +13,14 @@ import { getPendingSwaps } from '../../utils/queue';
 
 export class WidgetHistory {
   private manager: Manager | undefined;
+  private retrySwap: (pendingSwap: PendingSwap, meta: Meta) => void;
 
-  constructor(manager: Manager | undefined) {
+  constructor(
+    manager: Manager | undefined,
+    retrySwap: (pendingSwap: PendingSwap, meta: Meta) => void
```

Comment:
I think make the second parameters to a more general object would be better.
`actions: { retrySwap: .... }`. 
lets us to add/remove more easily in futute.

Reply:
fixed.